### ISSUE TITLE
Link to Documentation down

### ISSUE THREAD

**Issue by DeBelg** [waku-org/waku-rust-bindings#51]
The documentation redirection link on readMe file is offline.

**Comment by danielSanchezQ** [comment]
We have an issue building the docs actually. Problem is that it needs to build the project with go, which is not available easily in the docs runners. It is the first time I have this problem and do not know how to solve it actually.
@bacv When you have time, as this is not a priority, would you take a look at it?

**Comment by bacv** [comment]
> When you have time, as this is not a priority, would you take a look at it?

I'll take a look and report here. According to https://docs.rs/about/builds it should be possible if building docs doesn't require go-waku. Found some hints here: https://github.com/tensorflow/rust/blob/3f3f618678ffb42677366ee820d34b3772f18281/tensorflow-sys/Cargo.toml#L17-L19

**Comment by chair28980** [comment]
@bacv any updates on this issue? Is this still a pertinent issue?

**Comment by bacv** [comment]
I've pushed the changes required to the `docsrs/build-wo-go` but you'll need to test them when deploying on docs.rs environment.
Ideally the 4ceca08f4a7c36b24dc6790901c7b4ad9e7ec8c2 commit should work with `DOCS_RS` env var set, but when testing locally, the `proc-macro-error` crate fails to be built.
For this reason I've followed the previously linked tensorflow-sys example (code in 1edc69fff0a6581371b30722bfa53d426e44dba3). Both commits needs to be tested on docs.rs.
